__tests__/app.test.js

```diff
@@ -143,4 +143,51 @@ describe.only('GET /api/articles', () => {
 
         })
     })
+})
+
+describe('GET /api/articles/:article_id/comments', () => {
+    test('should respond with 200 for a good request', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({ body }) => {
+            
+            expect(Array.isArray(body.comments)).toBe(true)
+            if (body.comments.length !== 0) {
```

Comment:
There's no need to check if this is an array or if the length of the array is 0. If you are testing an article that does have comments, this will respond with the array of comments. Your test below this comment is great and also proves that it is an array